src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = true) {
+  const filterPeoples = people.filter(person =>
```

Comment:
use destructure assignment where it's possible
```suggestion
  const filterPeoples = people.filter(({ sex, died })=>
```